packages/plugins/plugin-registry/src/capabilities/app-graph-builder.ts

```diff
@@ -73,17 +84,17 @@ export default (context: PluginsContext) =>
                 testId: 'pluginRegistry.experimental',
               },
             },
-            {
-              id: `${REGISTRY_KEY}:community`,
-              type: 'category',
-              data: `${REGISTRY_KEY}:community`,
-              properties: {
-                label: ['community plugins label', { ns: REGISTRY_PLUGIN }],
-                icon: 'ph--users-three--regular',
-                key: REGISTRY_KEY,
-                testId: 'pluginRegistry.community',
-              },
-            },
+            // {
+            //   id: `${REGISTRY_KEY}:community`,
+            //   type: 'category',
+            //   data: `${REGISTRY_KEY}:community`,
+            //   properties: {
+            //     label: ['community plugins label', { ns: REGISTRY_PLUGIN }],
+            //     icon: 'ph--users-three--regular',
+            //     key: REGISTRY_KEY,
+            //     testId: 'pluginRegistry.community',
+            //   },
+            // },
```

Comment:
Is it important to leave the commented-out code or can we delete it?

Reply:
i think i'll delete it. i think it was more of a suggestion from josiah that we might have this soon?